sunpy/__init__.py

```diff
@@ -32,15 +32,18 @@ class UnsupportedPythonError(Exception):
 
 
 def _get_bibtex():
+    import textwrap
     # Set the bibtex entry to the article referenced in CITATION.rst
     citation_file = os.path.join(os.path.dirname(__file__), 'CITATION.rst')
 
     with open(citation_file, 'r') as citation:
-        refs = citation.read().split('@ARTICLE')[1:]
-        if len(refs) == 0:
-            return ''
-        bibtexreference = r"@ARTICLE{}".format(refs[0])
-    return bibtexreference
+        # Extract the first bibtex block:
+        ref = citation.read().partition(".. code:: bibtex\n\n")[2]
+        lines = ref.split("\n")
+        # Only read the lines which are indented
+        lines = lines[:[l.startswith("    ") for l in lines].index(False)]
+        ref = textwrap.dedent(''.join(lines))
```

Comment:
The formating of the bibtex is completely messed up using this - all in one line. And it's only getting the APJ paper, not JOSS. Good if that's what we want. Maybe this sorts it 
```suggestion
        ref = textwrap.dedent('\n'.join(lines))
```

Reply:
I am deliberately only getting the ApJ paper.